tests/suite/test_suite.py

```diff
@@ -107,13 +107,17 @@
     # code update:
     code_update.test_verify_quotes,
     code_update.test_add_node_with_bad_code,
-    governance_history.test_ledger_is_readable,
-    governance_history.test_tables_doc,
     # curve migration:
     reconfiguration.test_change_curve,
     recovery.test,
     # jwt
     jwt_test.test_refresh_jwt_issuer,
+    #
+    #
+    #
+    # Below tests should always stay last
```

Comment:
It might be useful to extend the comment explaining why this is the case

Reply:
For test_tables_doc, to goal is to collect all the tables the tests have used, parse the doc, and complain if they aren't mentioned. So naturally, to avoid missing tables, you want it last.
For the ledger is readable, it's the same idea, you want to test as much of it as possible.

Neither test will break if it's not last, but they will be less useful.

Reply:
I've added a comment to reflect this.